library/libraries.js

```diff
@@ -963,7 +963,7 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         npm: 'ember-source',
         test: function(win) {
             var ember = win.Ember || win.Em;
-            if (ember && ember.propertyDidChange) {
+            if (ember) {
```

Comment:
Is there a secondary check that we can include with `&&` to ensure that we’re not falsely detecting something else named `Ember` or `Em` on the `window`?

Reply:
@johnmichel I have updated the PR to add a secondary check